playbooks/common/openshift-cluster/std_include.yml

```diff
@@ -7,6 +7,10 @@
   tags:
   - always
 
+- include: sanity_checks.yml
+  tags:
```

Comment:
Are we removing tags?

Reply:
I'm fine if it will all be done in a single PR or if you want to punt.

Reply:
I see tags coming out as we break things apart.  And after we are all done with breaking stuff apart, we come back one last time and sweep up any remaining tags.